### PR TITLE
kata-deploy: Use `tomlq` to configure containerd

### PR DESCRIPTION
**Pull request by fidencio**
Please, see each commit description for better understanding of what we're doing.

### CODE REVIEW

File: tools/packaging/kata-deploy/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,9 @@ RUN \
 	mkdir -p ${DESTINATION} && \
 	tar xvf ${WORKDIR}/${KATA_ARTIFACTS} -C ${DESTINATION} && \
 	rm -f ${WORKDIR}/${KATA_ARTIFACTS} && \
-	apk del curl
+	apk del curl && \
+	apk --no-cache add jq py3-pip && \
+	pip install yq
```
Comment (stevenhorsman) [inline tools/packaging/kata-deploy/Dockerfile:25]: And this is a bit confusing - `tomlq` relies on `jq`, and is installed as part of the pip version of `yq` could we add this to the commit message at least so we/I don't get confused about this later?

**Review by stevenhorsman — APPROVED** [review]
This looks okay to me, but I've not tested on all the tomlq commands manually

File: tools/packaging/kata-deploy/Dockerfile (hunk 1 above)
Comment (fidencio) [reply tools/packaging/kata-deploy/Dockerfile:25]: Sure, I can do that!

**Review by beraldoleal — APPROVED** [review]
lgtm, thanks.

File: tools/packaging/kata-deploy/Dockerfile (hunk 1 above)
Comment (fidencio) [reply tools/packaging/kata-deploy/Dockerfile:25]: Done!

**Review by amshinde — APPROVED** [review]

**Pull request merged**